docs/content/kv_commands.md

```diff
@@ -25,3 +25,20 @@ $ wrangler kv create "new kv namespace"
     title: "new kv namespace",
 }
 ```
+
+### `list`
+
+Outputs a list of all KV namespaces associated with your account id.
+
+#### Usage
+
+``` sh
+$ wrangler kv list
```

Comment:
so, i'd imagine we do something similar to deletes for listing keys;

    $ wrangler kv list NAMESPACE_ID

yeah?

Reply:
correct.